airflow/models/dag.py

```diff
@@ -1189,7 +1189,7 @@ def clear_dags(
                           end_date=end_date,
                           only_failed=only_failed,
                           only_running=only_running,
-                          confirm_prompt=False,
+                          confirm_prompt=confirm_prompt,
```

Comment:
This should be false, we already check confirm_prompt on L1178 and set `do_it` accordingly

```suggestion
                          confirm_prompt=False,
```

Reply:
Done. Didn't read through the code well :)